x/data/msgs.go

```diff
@@ -39,26 +39,25 @@ func (m MsgAnchor) GetSignBytes() []byte {
 }
 
 func (m *MsgAttest) ValidateBasic() error {
-	for _, addr := range m.Attestors {
-		if _, err := sdk.AccAddressFromBech32(addr); err != nil {
-			return sdkerrors.ErrInvalidAddress.Wrap(err.Error())
-		}
+	if _, err := sdk.AccAddressFromBech32(m.Attestor); err != nil {
+		return sdkerrors.ErrInvalidAddress.Wrap(err.Error())
 	}
-	if m.Hash == nil {
-		return sdkerrors.ErrInvalidRequest.Wrap("hash cannot be empty")
+
+	for _, hash := range m.Hashes {
```

Comment:
we should also check that `len(m.Hashes) != 0`

Reply:
Added in https://github.com/regen-network/regen-ledger/pull/971/commits/0681137f97065067f952e43d4c70f9295c05d2ad